splice_machine/src/test/java/com/splicemachine/triggers/Trigger_Exec_Stored_Proc_IT.java

```diff
@@ -386,7 +386,7 @@ public void testStatementTriggerSysStoredProc() throws Exception {
         if (originalLevel.equals("INFO")) {
             newlevel = "WARN";
         }
-        tb.named("log_level_change").after().insert().on("S").statement().
+        tb.named("log_level_change").before().insert().on("S").statement().
```

Comment:
intentional change: the first DB-10968 changed this because SYSCS_SET_LOGGER_LEVEL wasn't executable in BEFORE , since SYSCS_SET_LOGGER_LEVEL was set to MODIFIES_SQL. Now it is back to NO_SQL, can be executed in BEFORE triggers.